src/main/java/org/fundacionjala/coding/nestor/ChildrenMovie.java

```diff
@@ -0,0 +1,18 @@
+package org.fundacionjala.coding.nestor;
+
+public class ChildrenMovie extends Movie {
+    private static double PRICE = 1.5;
+
+    public ChildrenMovie(String title) {
+        super(title, MovieType.CHILDRENS);
+    }
+
+    @Override
+    public double getPrice(int daysRented) {
+        if (daysRented > DAYS) {
+            PRICE += (daysRented - DAYS) * FACTOR;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)